agent/conf/agent.properties

```diff
@@ -202,6 +202,11 @@ hypervisor.type=kvm
 # Disable memory ballooning on vm guests for overcommit, by default overcommit
 # feature enables balloon and sets currentMemory to a minimum value.
 #
+# The time interval (in seconds) at which the balloon driver will get memory stats updates.
+# This is equivalent to Libvirt's --period parameter when using the dommemstat command.
+# Default value: 60
```

Comment:
thanks @joseflauzino 
can you update the default value to 0 in  agent/conf/agent.properties ?